.github/workflows/manual-build.yaml

```diff
@@ -0,0 +1,24 @@
+name: Manual Build Dispatch
+on: 
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to Build From'
+        required: true
+      tag:
+        description: 'Additional tag for the build (such as alpha, beta, etc.) - Optional'
+        default: ''
+
+jobs:
+  manual-build-and-publish-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.branch }}
+
+      - name: Get Current Date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y%m%d')"
+	run: echo "hello world done"
```

Comment:
Is this line intentional?

Reply:
yes, its a hello world I need to verify working and then I can add actual logic